src/archive.rs

```diff
@@ -203,12 +203,28 @@ impl<'a> Archive<'a> {
     }
 }
 
+bitflags! {
+    #[derive(Default)]
+    pub struct ArchiveFlags: u32 {
```

Comment:
Do you need to make this `pub` if you're not exposing the `flags` field in `OpenArchive`?

Reply:
No, unless it's moved over to `unrar_sys`. Will fix that.